latest/docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -43,35 +43,19 @@ az login -u <username> -p <password>
 
 Service principals are like user accounts to which you can apply rules using Azure Active Directory.
 Authenticating with a service principal is the best way to secure the usage of your Azure resources
-from either your scripts or applications that manipulate resources.
-You define the roles you want your users to have via the `az role` set of commands.
-You can learn more and see examples of service principal roles in our [az role reference articles](https://docs.microsoft.com/cli/azure/role.md).
-
-1. If you don't already have a service principal, [create one](create-an-azure-service-principal-azure-cli.md).
-
-1. Log in with the service principal.
-
-   ```azurecli-interactive
-   az login --service-principal -u "http://my-app" -p <password> --tenant <tenant>
-   ```
-
-   To get your tenant, log in interactively and then get the tenantId from your subscription.
-
-   ```azurecli
-   az account show
-   ```
-
-   ```json
-   {
-       "environmentName": "AzureCloud",
-       "id": "********-****-****-****-************",
-       "isDefault": true,
-       "name": "Pay-As-You-Go",
-       "state": "Enabled",
-       "tenantId": "********-****-****-****-************",
-       "user": {
-       "name": "********",
-       "type": "user"
-       }
-   }
-   ```
+from either your scripts or applications that manipulate resources. If you don't already have a service principal available and
+would like to create one, see [Create an asure service principal with Azure CLI](create-an-azure-service-principal-azure-cli.md)
+
+To log in with a service principal, you provide the username, password or certificate PEM file, and the tenant associated with the service principal:
+
+```azurecli-interactive
+az login --service-principal -u <user> -p <password-or-cert> --tenant <tenant>
+```
+
+The tenant value is the Azure Active Directory tenant associated with the service principal. This can either be an .onmicrosoft.com domain, or the Azure object ID for the tenant.
+You can get the tenant object ID for your current login by using the following command:
+
+```azurecli
+az account show --query 'tenanatId' -o tsv
```

Comment:
In some places we use `-o` and other `--output`.  I don't *think* this is an issue, as I would like people to know that you can shorten it.